site/.vitepress/config.ts

```diff
@@ -29,6 +42,10 @@ export default defineConfig({
             text: "Package Overview",
             link: "/package-overview",
           },
+          {
+            text: "Demo",
+            link: "https://gasless-nft-minter.vercel.app/",
+          },
```

Comment:
Can you add my example too? https://aa-simple-dapp.vercel.app

```suggestion
          {
            text: "NFT Gas-less Minting Demo",
            link: "https://gasless-nft-minter.vercel.app/",
          },
          {
            text: "Token Gas-less Minting Demo",
            link: "https://aa-simple-dapp.vercel.app/",
          },
```

Reply:
ah yea can do